_templates/makeRule/new_rule/prompt.js

```diff
@@ -0,0 +1,25 @@
+// see types of prompts:
+// https://github.com/enquirer/enquirer/tree/master/examples
+//
+module.exports = [
+  {
+    type: 'input',
+    name: 'start_date_YYYY',
+    message: "Start Year (Format: YYYY):"
+  },
+  {
+    type: 'input',
+    name: 'dataset_name',
+    message: "Dataset Name (Example: WV04_MSI_L1B):"
+  },
+  {
+    type: 'input',
+    name: 'version_str',
+    message: "Version Number (Example: 1):"
+  },
+  {
+    type: 'input',
+    name: 'provider_path_format',
+    message: "providerPathFormat (Example: 'css/nga/WV03/1B/'yyyy/DDD):"
+  }
+]
```

Comment:
I suggest a few tweaks here:

1. rename "dataset" to "collection", as that's the term used throughout
2. reorder prompts to be somewhat form broader to narrower scope
3. use camelCase for var names (e.g., collectionName rather than collection_name), as that's the JavaScript convention

I also suggest changing the path of this file from `_templates/makeRule/new_rule/prompt.js` to `_templates/rule/new/prompt.js` so the hygen command to use this is simply `hygen rule new` rather than `hygen makeRule new_rule`, which is slightly redundant and slightly more verbose.

```suggestion
module.exports = [
  {
    type: 'input',
    name: 'provider',
    message: "Provider ID:"
  },
  {
    type: 'input',
    name: 'collectionName',
    message: "Collection name:"
  },
  {
    type: 'input',
    name: 'collectionVersion',
    message: "Collection version:"
  },
  {
    type: 'input',
    name: 'providerPathFormat',
    message: "Provider path format (example: 'css/nga/WV03/1B/'yyyy/DDD):"
  },
  {
    type: 'input',
    name: 'year',
    message: "Year (YYYY):"
  }
]
```